static/src/javascripts/bootstraps/enhanced/main.js

```diff
@@ -384,6 +385,19 @@ const bootEnhanced = (): void => {
                 );
             }
 
+            if (config.get('page.contentType') === 'Audio') {
+                require.ensure(
+                    [],
+                    require => {
+                        bootstrapContext(
+                            'audio',
+                            require('common/modules/audio').init
```

Comment:
That module doesn't seem to exist.

Reply:
It does! that's how the player code turns up on the page